local_gemma/cli.py

```diff
@@ -91,6 +91,13 @@
     type=str,
     help="Forces a specific device to be used. By default uses cuda > mps > cpu, depending on availability.",
 )
+parser.add_argument(
+    "--quantization",
+    type=str,
+    choices=["bnb", "hqq"],
```

Comment:
Let's also allow users to pass `quanto` and fully select between the different quant options?